internal/service_helper.go

```diff
@@ -38,5 +39,9 @@ func GetHeadlessService(cluster *fdbv1beta2.FoundationDBCluster) *corev1.Service
 	service.Spec.ClusterIP = "None"
 	service.Spec.Selector = cluster.GetMatchLabels()
 
+	if cluster.Spec.Routing.PodIPFamily != nil && *cluster.Spec.Routing.PodIPFamily == 6 {
```

Comment:
This will not work, see: https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services

```text
Note: The .spec.ipFamilies field is immutable because the .spec.ClusterIP cannot be reallocated on a Service that already exists. If you want to change .spec.ipFamilies, delete and recreate the Service.
```

Reply:
Here we are just creating a service type not the the actual service, the actual call for service creation is done in addService reconciler.

Reply:
done